docker-compose.yml

```diff
@@ -0,0 +1,18 @@
+version: '3'
+
+services:
+  db:
+    image: mdillon/postgis:9.4
+    ports:
+      - "5432"
+  ashlar-server:
+    build: ./ashlar-server
+    links:
```

Comment:
It looks like `links:` is not exactly deprecated, but discouraged in v3 docker-compose files: https://docs.docker.com/compose/compose-file/#links . I think that `depends_on` should be enough here.

Reply:
True, I don't think it's doing anything either. I'll get rid of it.